recipes/mapdamage2/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: mapdamage2
+  version: "2.0.6"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: 2.0.6.tar.gz
+  url: https://github.com/ginolhac/mapDamage/archive/2.0.6.tar.gz
+  md5: f14e1ca52d68fd9c7e42e1f4e6f40f8c
+
+requirements:
+  build:
+    - python >=2.7,<3
+    - zlib
+  run:
+    - python >=2.7,<3
+    - pysam
+    
+# Skip test for now due to error
+# Error: Could not import required module 'pysam':
+# 	- /lib64/libc.so.6: version `GLIBC_2.7' not found (required by /anaconda/envs/_build/lib/python2.7/site-packages/pysam/../../../libhts.so.1)
+#test:
```

Comment:
Submitting without tests is not allowed, please try to get this test working.
If you depend on GCC and include a libgcc as runtime dependency it might work.